guacamole/src/main/webapp/translations/nl.json

```diff
@@ -43,7 +45,7 @@
     "CLIENT" : {
 
         "ACTION_ACKNOWLEDGE"               : "@:APP.ACTION_ACKNOWLEDGE",
-        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Wis lijst",
+        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Wis lijst Voltooide Overdrachten",
```

Comment:
Is "Wis lijst" incorrect? Is there a shorter way to say this than "Wis lijst Voltooide Overdrachten"? The new user interface will not accommodate such a large button very well. The English translation, for example, is simply "Clear".